kubernetes/couchdb/docker/init.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -ex
+
+# Always clone the latest version of OpenWhisk
+git clone https://github.com/apache/incubator-openwhisk /openwhisk
+
+pushd /openwhisk
+  # Install ansible requirements
+  ./tools/ubuntu-setup/pip.sh
+
+  # upgrade cffi for ansible error on Debian Jesse
+  pip install --upgrade cffi
+  sudo pip install markupsafe
+  sudo pip install ansible==2.3.0.0
+
+
+  # if auth guest overwrite file
+  if [ -n "$AUTH_GUEST" ]; then
+    echo "$AUTH_GUEST" > /openwhisk/ansible/files/auth.guest
+  fi
+
+  # if auth whisk system overwrite file
+  if [ -n "$AUTH_WHISK_SYSTEM" ]; then
+    echo "$AUTH_WHISK_SYSTEM" > /openwhisk/ansible/files/auth.guest
```

Comment:
This should be: 

echo "$AUTH_WHISK_SYSTEM" > /openwhisk/ansible/files/auth.whisk.system